data/890/455/647/890455647.geojson

```diff
@@ -392,10 +392,11 @@
     "wof:belongsto":[
         102191569,
         85632573,
-        85667951,
-        1376833605
+        1376833605,
+        85667951
     ],
     "wof:breaches":[],
+    "wof:capital_of":85667951,
```

Comment:
Another example of needing to be in a list.

Reply:
Fixed in 8d898b1.

Reply:
This was fixed in 56a8ef4.